client/src/components/_custom/drp/utils/utils.js

```diff
@@ -5,6 +5,20 @@ export const formatLabel = (key) => {
     if (typeof key !== 'string') {
         return key;
     }
+
+    // Check if it's a valid URL
+    const isValidUrl = (str) => {
+        try {
+            const url = new URL(str);
+            return url.protocol === 'http:' || url.protocol === 'https:';
+        } catch {
+            return false;
+        }
+    };
+
+    if (isValidUrl(key)) {
+        return key;
+    }
```

Comment:
```suggestion
return isValidUrl(key);
```

Consider being less verbose, for clarity.